### PR TITLE
Issue #4978: renamed sub-children filters to filter element

### PR DESCRIPTION
**Pull request by rnveach**
Issue #4978

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java [hunk 1]
```diff
@@ -827,12 +827,12 @@ private static void fillModulesFromFilefiltersPackage() {
...
-        NAME_TO_FULL_MODULE_NAME.put("IntMatchFilter",
-                BASE_PACKAGE + ".filters.IntMatchFilter");
-        NAME_TO_FULL_MODULE_NAME.put("IntRangeFilter",
-                BASE_PACKAGE + ".filters.IntRangeFilter");
+        NAME_TO_FULL_MODULE_NAME.put("CsvFilterElement",
+                BASE_PACKAGE + ".filters.CsvFilterElement");
+        NAME_TO_FULL_MODULE_NAME.put("IntMatchFilterElement",
+                BASE_PACKAGE + ".filters.IntMatchFilterElement");
+        NAME_TO_FULL_MODULE_NAME.put("IntRangeFilterElement",
+                BASE_PACKAGE + ".filters.IntRangeFilterElement");
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java]: These should not be listed here. Most are package-private and they don't extend AutomaticBean.
I will make another PR after this one to remove them.

**Comment by romani** [comment]
Ok to merge

**Comment by rnveach** [comment]
pitest failure was because suppression file had class names in it that needed to be changed too.

**Comment by rnveach** [comment]
> `-PackageObjectFactory.java.html:<td class='covered'><pre><span  class='survived'>        if (!name.contains(PACKAGE_SEPARATOR)) {</span></pre></td></tr>`

I don't understand how a file rename caused a mutation to be killed.

**Comment by rnveach** [comment]
@romani pitest1 passed without changing anything and just re-running CI.
I think we may need to look into splitting up pitest more.

**Comment by rnveach** [comment]
Since CI has passed, I will merge this.

**Pull request merged**